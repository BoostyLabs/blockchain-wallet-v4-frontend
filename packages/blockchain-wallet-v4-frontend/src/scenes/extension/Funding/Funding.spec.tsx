import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import { Funding } from '.'

it('renders correctly', () => {
  const component = shallow(<Funding />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
