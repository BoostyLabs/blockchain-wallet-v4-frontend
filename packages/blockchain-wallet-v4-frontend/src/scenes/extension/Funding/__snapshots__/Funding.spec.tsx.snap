// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<styled.section>
  <styled.div
    onClick={[Function]}
  >
    <Memo()
      color="white060"
      label="Icon Activity"
      size="md"
    >
      <SvgIconArrowLeft />
    </Memo()>
  </styled.div>
  <SelectAccount
    setFundingStep={[Function]}
  />
</styled.section>
`;
